src/components/LandmarkDrawer.tsx

```diff
@@ -5,19 +5,41 @@ import {
     DrawerOverlay, 
     DrawerContent, 
     DrawerCloseButton,
+    Radio,
+    RadioGroup,
+    Stack,
+    Text,
+    Button,
   } from '@chakra-ui/react';
 import { Landmark } from 'shared/Map';
+import React, { useState } from 'react';
+import { componentSpacing } from 'theme/metrics';
 
 export default function LandmarkDrawer ({ onClose, landmark }: { 
     onClose: () => void; 
     landmark: Landmark
   }) {      
-    return <Drawer isOpen onClose={onClose}>
+    const [value, setValue] = useState('');
+
+    return <Drawer size="lg" isOpen onClose={onClose}>
       <DrawerOverlay />
       <DrawerContent>
         <DrawerCloseButton /> 
         <DrawerHeader>{landmark.名称}</DrawerHeader>
-        <DrawerBody>{landmark.定义}</DrawerBody>
+        <DrawerBody display="flex" flexDirection="column" gap={componentSpacing}>
+          {landmark.定义}
+          <Text>你认为你的{landmark.名称}处于以下哪个阶段？（单选）</Text>
+          <RadioGroup onChange={setValue} value={value}>
+            <Stack direction="column">
+              {landmark.层级.map((level, index) => (
+                <Radio key={index} value={String(index)} alignItems="flex-start">
```

Comment:
why flex-start?

Reply:
The paragraph is too long hence the button will appear in the middle. Which one do you prefer?
![Screenshot 2024-09-23 at 17 49 39](https://github.com/user-attachments/assets/18548d8c-cd48-438a-b119-ee5499f0f837)